tests/wast.rs

```diff
@@ -187,7 +187,15 @@ fn ignore(test: &Path, strategy: Strategy) -> bool {
 
         if part == "typed-continuations" {
             // TODO(dhil): Tag linking is currently broken
-            return test.ends_with("linking_tags.wast");
+            if test.ends_with("linking_tags.wast") {
+                return true;
+            }
+
+            // This test specifically checks that we catch a continuation being
+            // resumed twice, which we cannot detect in this mode.
+            if test.ends_with("cont_twice.wast") {
+                return true;
+            }
```

Comment:
Surely this is only true when `unsafe_disable_continuation_linearity_check` is toggled?

Reply:
Yes, that check should read` test.ends_with("cont_twice.wast") && cfg!(feature = "unsafe_disable_continuation_linearity_check")`.

My intention is to make sure that the test suite passes normally with this feature enabled, thus disabling this particular test in its presence. Given that, I had to remove the check regarding `cont_twice.wast` from `main.yml`. Or are you particularly interested in ensuring that the test _does_ indeed fail if `unsafe_disable_continuation_linearity_check` is enabled? That's reasonable (the test case will just not trap with the message expected in that case), but beyond that I would consider the behaviour of that program to be undefined.